### ISSUE TITLE
Hyperparameters problem

### ISSUE THREAD

**Issue by tzffm** [gzerveas/mvts_transformer#73]
Hello, you split the training set into 0.8 training set and 0.2 validation set. In the classification task, do you adjust the hyperparameters based on the minimum loss or the maximum accuracy？ Thank you.
